appointment-svc/controllers/appointment-controller.ts

```diff
@@ -20,6 +20,25 @@ class AppoinmentController {
     }
   }
 
+  async getAvailableAppointment(
+    req: Request,
+    res: Response,
+    next: NextFunction
+  ) {
+    let queryCurrentDate = req.query.currentDate as string;
+    let currentDate = new Date(queryCurrentDate);
+    let targetDate = new Date(currentDate);
+    targetDate.setDate(currentDate.getDate() + 1);
```

Comment:
I think you have made a mistake here. You declare targetDate, and I think you never use it. You don't pass it to the getAvailableAppointment instance, and you just declare it again.

Reply:
good catch. i forgot to delete it. i initially had it so that it accepted two arguments (current date and target date) but decided to refactor it so that it only accepted one argument (current date). it will then calculate the target date inside the function (current date + 1 day).